README.rst

```diff
@@ -68,6 +84,17 @@ Asynchronous `scroll <https://www.elastic.co/guide/en/elasticsearch/reference/cu
     loop.run_until_complete(go())
     loop.close()
 
+Running the tests
+-----------------
+
+Pytests
+
+.. code-block:: python
+
+    python -m pytest 
```

Comment:
pytest tests

Reply:
I'd put test path into `pytest.ini` rather than forcing the user to do this manually